x-pack/plugins/cloud_security_posture/server/saved_objects/saved_objects.ts

```diff
@@ -6,34 +6,14 @@
  */
 
 import { SavedObjectsServiceSetup } from '@kbn/core/server';
-import { SECURITY_SOLUTION_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
-import { cspRuleTemplateSavedObjectMapping } from './mappings';
-import { cspRuleTemplateMigrations } from './migrations';
-import {
-  cspRuleTemplateSchemaV830,
-  cspRuleTemplateSchemaV840,
-  cspRuleTemplateSchemaV870,
-  CspRuleTemplate,
-} from '../../common/schemas';
 
-import { CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE } from '../../common/constants';
+import { CspSettings } from '../../common/schemas/csp_settings';
+import { CspRuleTemplate } from '../../common/schemas';
+
+import { cspRuleTemplate } from './csp_rule_template';
+import { cspSettings } from './csp_settings';
 
 export function setupSavedObjects(savedObjects: SavedObjectsServiceSetup) {
-  savedObjects.registerType<CspRuleTemplate>({
-    name: CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE,
-    indexPattern: SECURITY_SOLUTION_SAVED_OBJECT_INDEX,
-    hidden: false,
-    namespaceType: 'agnostic',
-    management: {
-      importableAndExportable: true,
-      visibleInManagement: true,
-    },
-    schemas: {
-      '8.3.0': cspRuleTemplateSchemaV830,
-      '8.4.0': cspRuleTemplateSchemaV840,
-      '8.7.0': cspRuleTemplateSchemaV870,
-    },
-    migrations: cspRuleTemplateMigrations,
-    mappings: cspRuleTemplateSavedObjectMapping,
-  });
+  savedObjects.registerType<CspRuleTemplate>(cspRuleTemplate);
+  savedObjects.registerType<CspSettings>(cspSettings);
```

Comment:
update the file to follow [Kibana core team requirements](https://github.com/elastic/kibana/blob/main/dev_docs/tutorials/saved_objects.mdx#L12).